### PR TITLE
Editor: Sync packages to include gutenberg_ to wp_ prefix change

### PR DESCRIPTION
**Pull request by ockham**
To include [this fix](https://github.com/WordPress/gutenberg/pull/51989).

Trac ticket: https://core.trac.wordpress.org/ticket/58651

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,12 +90,12 @@
 		"@wordpress/block-serialization-default-parser": "4.35.1",
 		"@wordpress/blocks": "12.12.1",
 		"@wordpress/commands": "0.6.3",
-		"@wordpress/components": "25.1.3",
+		"@wordpress/components": "23.9.0",
```
Comment (spacedmonkey) [inline package.json]: This seems wrong?

**Comment by ockham** [comment]
I think I need to temporarily remove `@wordpress/nux` before running the sync script (`npm run sync-gutenberg-packages -- --dist-tag=wp-6.3`) (and then add it back) to get all the right version updates. Will do that now.

File: package.json (hunk 1 above)
Comment (ockham) [reply package.json]: Yeah, I made a mistake, see https://github.com/WordPress/wordpress-develop/pull/4748#issuecomment-1612157261. Should be fixed now.

**Comment by ockham** [comment]
> I think I need to temporarily remove `@wordpress/nux` before running the sync script (`npm run sync-gutenberg-packages -- --dist-tag=wp-6.3`) (and then add it back) to get all the right version updates. Will do that now.

Done.

**Review by joemcgill — APPROVED** [review]
Looks better to me

**Comment by sabernhardt** [comment]
https://core.trac.wordpress.org/ticket/58651

**Review by spacedmonkey — APPROVED** [review]
:shipit: Ship it.

**Review by felixarntz — APPROVED** [review]

**Comment by spacedmonkey** [comment]
@ockham CI seems to be failing :(

**Comment by ockham** [comment]
> @ockham CI seems to be failing :(

Dang. I think I need to `npm run build:dev` to update the script loader file...

**Comment by ockham** [comment]
Committed to Core `trunk` in https://core.trac.wordpress.org/changeset/56098.

**Pull request closed without merging**